.bash_profile

```diff
@@ -0,0 +1,8 @@
+PATH="Library/Frameworks/Python.framework/Versions/3.5/bin:${PATH}"
```

Comment:
please add these files to .gitignore files and remove them from the pull request, ditto below